activities/supporting-codebase-governance/technical-roadmap-template.md

```diff
@@ -0,0 +1,58 @@
+---
+tags: Templates
+---
+
+# Technical Roadmap Template
+
+This is a template that can be used for technical roadmaps, it's based on a version that [Sharetribe used](https://github.com/sharetribe/sharetribe/blob/a029d02ab3b60009ebb6faff54117ba1034a6bfc/TECHNICAL_ROADMAP.md).
+
+## Overview
+
+This document sheds light on the current development plans of the team.
+Plans change constantly as new information comes to light.
+Thus, this document represents **a snapshot of our plans at this point in time**, based on the information that is currently available.
+
+In other words, the purpose of this document is to share the current state of the team's understanding.
+
+To reiterate: the plans shared in this document should not be considered set in stone.
+They will change when relevant information is absorbed by the team.
+
+## What is currently being worked on
+
+*Not all sections need to be filled in, but the title and why are recommended.*
+
+### Technical thing 1
+
+#### Why 1
+
+#### How 1
+
+#### What 1
+
+### Technical thing 2
+
+#### Why 2
+
+#### How 2
+
+#### What 2
```

Comment:
Are these supposed to be limited to two? Or should they be for whatever thing is worked on? Giving numberings implies it is limited.

I suggest adding this to the guidance provided.

Als I suggest removing the emphasis here, however, that's just a suggestion.

```suggestion
*Not all sections need to be filled in, but the title and why are recommended.*
You can add new items by adding a heading for the title, why, how, and what.

### [Technical thing name or title]

#### Why

#### How

#### What
```